diff -uNr oslo_log.orig/log.py oslo_log/log.py
--- oslo_log.orig/log.py	2015-08-24 21:33:51.000000000 +0000
+++ oslo_log/log.py	2015-11-16 20:01:42.982377423 +0000
@@ -337,10 +337,10 @@
         facility = _find_facility(conf.syslog_log_facility)
         # TODO(bogdando) use the format provided by RFCSysLogHandler after
         # existing syslog format deprecation in J
-        syslog = handlers.OSSysLogHandler(
+        syslog_handler = handlers.OSSysLogHandler(
             facility=facility,
             use_syslog_rfc_format=conf.use_syslog_rfc_format)
-        log_root.addHandler(syslog)
+        log_root.addHandler(syslog_handler)
 
     datefmt = conf.log_date_format
     for handler in log_root.handlers:

